Index: src/main/java/com/digital/aida/HelloName.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package com.digital.aida;\n\npublic class HelloName {\n    public String HelloName(String name) {\n        return \"Hello\" + name + \"!\";\n\n    }\n}\n\n\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/com/digital/aida/HelloName.java b/src/main/java/com/digital/aida/HelloName.java
--- a/src/main/java/com/digital/aida/HelloName.java	(revision 733b36b5c1769e8b86e768fe12fe4d807dbc5c92)
+++ b/src/main/java/com/digital/aida/HelloName.java	(date 1697466597093)
@@ -5,6 +5,5 @@
         return "Hello" + name + "!";
 
     }
-}
+    }
 
-
