Index: src/main/java/com/digital/aida/HelloName.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package com.digital.aida;\n\npublic class HelloName {\n    public String HelloName(String name) {\n        return \"Hello\" + name + \"!\";\n\n    }\n    }\n\n<<<<<<< HEAD\n=======\n\n>>>>>>> origin\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/com/digital/aida/HelloName.java b/src/main/java/com/digital/aida/HelloName.java
--- a/src/main/java/com/digital/aida/HelloName.java	(revision 1ddddbeb3f1628dc4a4ff3831140b9bf292bcc89)
+++ b/src/main/java/com/digital/aida/HelloName.java	(date 1697466545522)
@@ -7,7 +7,3 @@
     }
     }
 
-<<<<<<< HEAD
-=======
-
->>>>>>> origin
