Index: src/main/java/com/digital/sergey/HelloName.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package com.digital.sergey;\r\n\r\npublic class HelloName {\r\n\r\n    public String helloName(String name) {\r\n        return \"Hello\"+\" \"+name+\"!\";\r\n\r\n    }\r\n\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/com/digital/sergey/HelloName.java b/src/main/java/com/digital/sergey/HelloName.java
--- a/src/main/java/com/digital/sergey/HelloName.java	(revision ef4e77b5fa56750b96cd18718ff0d19d856bc76f)
+++ b/src/main/java/com/digital/sergey/HelloName.java	(date 1697442341710)
@@ -7,4 +7,6 @@
 
     }
 
+
+
 }
