Index: src/main/java/com/digital/sergey/HelloName.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package com.digital.sergey;\r\n\r\npublic class HelloName {\r\n\r\n    public String helloName(String name) {\r\n        return \"Hello\"+\" \"+name+\"!\";\r\n\r\n    }\r\n\r\n\r\n\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/com/digital/sergey/HelloName.java b/src/main/java/com/digital/sergey/HelloName.java
--- a/src/main/java/com/digital/sergey/HelloName.java	(revision 2e41b0c6be7d3d6fea29afa810d926965fc02a44)
+++ b/src/main/java/com/digital/sergey/HelloName.java	(date 1697442082743)
@@ -6,7 +6,4 @@
         return "Hello"+" "+name+"!";
 
     }
-
-
-
 }
